packages/comet-extras/src/components/toast/toast.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import { Meta, StoryFn } from '@storybook/react';
+import Toast, { ToastProps } from './toast';
+import Button from '../../../../comet-uswds/src/components/button/button';
+import ButtonGroup from '../../../../comet-uswds/src/components/button-group/button-group';
+
+const meta: Meta<typeof Toast> = {
+  title: 'Extras/Toast',
+  component: Toast,
+  argTypes: {
+    id: { control: false },
+    message: { control: 'text' },
+    duration: { control: 'number' },
+    type: { control: false },
```

Comment:
This should be set to a `select` control like here: https://github.com/MetroStar/comet/blob/9eb0da76bc6d985a1b2aac08cd5fef1a7fa04c1c/packages/comet-uswds/src/components/alert/alert.stories.tsx#L11